configs/default/policy.yaml

```diff
@@ -0,0 +1,27 @@
+#
+# To use this configuration:
+#   ec validate image ... --policy github.com/enterprise-contract//configs/default.yaml?ref=main
+#
+description: >
+  Use the policy rules from the "minimal" collection. This and other collections are defined in
+  https://redhat-appstudio.github.io/docs.stonesoup.io/ec-policies/release_policy.html#_available_rule_collections
```

Comment:
Should this URL and the one below go to enterprisecontract.dev?

Reply:
Done.